app/src/main/java/com/example/recipeapp/Models/Recipe.java

```diff
@@ -34,37 +49,28 @@ public class Recipe {
     List<InstructionEnvelope<List<Step>>> analyzedInstructions;
     @SerializedName("extendedIngredients")
     List<Ingredient> ingredients;
+    @SerializedName("cuisines")
+    List<String> cuisines;
+    @SerializedName("diets")
+    List<String> diets;
 
-    public static Recipe fromJson(JSONObject object)
-    {
-        Recipe recipe = new Recipe();
-        try {
-            recipe.title = object.getString("title");
-            recipe.image = object.getString("image");
-            recipe.id = object.getLong("id");
-            recipe.readyInMinutes = object.getInt("readyInMinutes");
-            recipe.servings = object.getInt("servings");
-        } catch (JSONException e) {
-            e.printStackTrace();
-        }
-        return recipe;
+    @Transient
+    double userRating = -3;
```

Comment:
Any context why we pick `-3` here? Does this context worth a in code comment?

Reply:
Lets add const to the number as well as docblock as to why this number is being added